Library/brew.sh

```diff
@@ -1,20 +1,7 @@
 HOMEBREW_VERSION="0.9.5"
 
-odie() {
-  if [[ -t 2 ]] # check whether stderr is a tty.
-  then
-    echo -ne "\033[4;31mError\033[0m: " >&2 # highlight Error with underline and red color
-  else
-    echo -n "Error: " >&2
-  fi
-  if [[ $# -eq 0 ]]
-  then
-    /bin/cat >&2
-  else
-    echo "$*" >&2
-  fi
-  exit 1
-}
+# shellcheck source=/dev/null
```

Comment:
`shellcheck` complains that it cannot check the sourced file because it cannot find it (because the path is using some variable and its value is only known at run time). In theory, this could be:

```
# shellcheck source=Library/Homebrew/utils.rb
```

But then `shellcheck` will print an even more confusing warning that it can't find the file, unless one always makes sure to run `shellcheck` with `$HOMEBREW_REPOSITORY` as the current working directory.